client/core/core.go

```diff
@@ -218,7 +218,10 @@ func (dc *dexConnection) hasActiveOrders() bool {
 		if checkMatchOrderStatus(trade) {
 			return true
 		}
-		if trade.metaData.Status == order.OrderStatusBooked || trade.metaData.Status == order.OrderStatusEpoch {
+		//trade.mtx.RLock() // ?
+		status := trade.metaData.Status
+		//trade.mtx.RUnlock()
+		if status == order.OrderStatusBooked || status == order.OrderStatusEpoch {
```

Comment:
The purpose of the `trackedTrade.mtx` field unclear.  I think it needs documentation and a review of the methods and call stack to be sure it is locked as require. @itswisdomagain